circuitbreakerrpc/circuitbreaker.proto

```diff
@@ -40,6 +40,12 @@ service Service {
             get:"/limits"
         };        
     }
+
+    rpc ListForwardingHistory (ListForwardingHistoryRequest) returns (ListForwardingHistoryResponse) {
+        option (google.api.http) = {
+            get:"/forwarding_history"
```

Comment:
I think the start and end time need to be mapped to query params here?

Reply:
Looks like it's built in? 

I can query: `http://localhost:9235/api/forwarding_history?add_start_time_ns=1695052103587689000` with no changes to the proto and it filters accordingly.

Reply:
Ok good! int64 apparently is hard to use for javascript people, but also don't know how to do it in another way. I assume the returned data needs sub-second precision, so regular unix timestamp isn't suitable for the response. Using it here would be inconsistent.